hangmanWeb.js

```diff
@@ -0,0 +1,126 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Overall very good work! I liked frequent and proper use of functions to minimize DRY codes. One feedback is CSS and design choice. Position of "_ _ _ _ _" line is way at the bottom which makes it hard to see without scrolling al the way to the bottom. Also "You Lost" message cannot be seen without scrolling down as well. Other than design feedback, great job with project! I'm looking forward to see more awesome projects in the future.